CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 1.1.7 (2021-10-13)
+## iotedged
+### Bug Fixes
+* Fix a vulnerability in the `allow_privileged` flag [6725a11](https://github.com/Azure/iotedge/commit/6725a11dc6a557478d9fe11820847e75f4ab27a1)
```

Comment:
It looks like the flag is actually named `allow_elevated_docker_permissions` not `allow_privileged`.